lib/marked.js

```diff
@@ -762,8 +703,18 @@ InlineLexer.prototype.output = function(src) {
       src = src.substring(cap[0].length);
       this.inLink = true;
       href = cap[2];
-      href = href[0] === '<' ? href.substring(1, href.length - 1) : href;
-      title = cap[3] ? cap[3].substring(1, cap[3].length - 1) : cap[3];
+      if (this.options.pedantic) {
+        link = /^([^'"(]*[^\s])\s+(['"(].*['")])/.exec(href);
+
+        if (link) {
+          href = link[1];
+          title = link[2].trim().slice(1, -1);
```

Comment:
If you change the regex to `...\s+['"(](.*)['")]` then you wouldn't need this slice.

Reply:
I added `(` and `)` to the list of valid ways to denote a title based on the CommonMark [spec](https://spec.commonmark.org/0.28/#link-title). They were not in the original regex. Do we have a test for this?